charts/seed-monitoring/charts/grafana/dashboards/operators/apiserver-watches-dashboard.json

```diff
@@ -0,0 +1,391 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "editable": true,
+  "gnetId": null,
+  "graphTooltip": 1,
+  "id": 22,
+  "iteration": 1597754908762,
+  "links": [],
+  "panels": [
+    {
+      "datasource": null,
+      "gridPos": {
+        "h": 1,
+        "w": 24,
+        "x": 0,
+        "y": 0
+      },
+      "id": 4,
+      "panels": [],
+      "repeat": "Scope",
+      "scopedVars": {
+        "Scope": {
+          "selected": true,
+          "text": "cluster",
+          "value": "cluster"
+        }
+      },
+      "title": "Watch Durations for Scope: $Scope",
+      "type": "row"
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "description": "$Quantile of watches on this resource are less than or equal to the value.",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {}
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 11,
+        "w": 24,
+        "x": 0,
+        "y": 1
+      },
+      "hiddenSeries": false,
+      "id": 2,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "dataLinks": []
+      },
+      "percentage": false,
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "repeat": "Quantile",
```

Comment:
IIUC, this configures to have one panel per selected quantile, right?
I think for `scope` it makes kind of sense, but it would be nice to be able to see the different quantiles in one panel. (maybe also for `scope`)
WDYT?

Reply:
I can remove the repeat for `Quantile`, but I would suggest that we keep the repeat for `scope`. At some point the panel will become too crowded and I think separating it by scope is fine. There is a shared tooltip that makes it easy to compare different panels.

Reply:
Yes, sounds good 👍